packages/wonder-blocks-data/src/__tests__/generated-snapshot.test.js

```diff
@@ -104,27 +101,6 @@ describe("wonder-blocks-data", () => {
                         }}
                     </Data>
                 </View>
-                <Strut size={Spacing.small_12} />
-                <View>
-                    <Body>This cache has error!</Body>
-                    <Data handler={myHandler} requestId="ERROR">
-                        {(result) => {
-                            if (result.status !== "error") {
-                                return "If you see this, the example is broken!";
-                            }
-
-                            return (
-                                <BodyMonospace
-                                    style={{
-                                        color: Color.red,
-                                    }}
-                                >
-                                    ERROR: {result.error.message}
-                                </BodyMonospace>
-                            );
-                        }}
-                    </Data>
-                </View>
```

Comment:
This started failing and now because the request fulfillment no longer turns handler thrown errors into promises (which was delaying an error until after the snapshot render). It think the new behavior is more correct and so deleted this part of the example rather than trying to make it work.